src/App.css

```diff
@@ -1,23 +1,94 @@
 .App {
   display: grid;
   grid-template-rows: [header] 80px [main] auto;
-  gap: 10px;
+
   color: #04b1c5;
 
   width: 100vw;
   background-color: #111440;
-  font-size: 2em;
+  font-size: 16px;
 }
 
 .Header {
   grid-row: header;
-  display: flex;
+
   align-items: center;
   justify-content: center;
```

Comment:
align-items and justify-content belong to the flex. Without a flex they could be deleted